account_id.go

```diff
@@ -48,57 +48,62 @@ type _AccountIDs struct { //nolint
 // AccountIDFromString constructs an AccountID from a string formatted as
 // `Shard.Realm.Account` (for example "0.0.3")
 func AccountIDFromString(data string) (AccountID, error) {
-	shard, realm, num, checksum, alias, aliasEvmAddress, err := _AccountIDFromString(data)
+	shard, realm, num, checksum, alias, AliasEvmAddress, err := _AccountIDFromString(data)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏For consistency `AliasEvmAddress` should be in camel case as the rest of the variables.